### PR TITLE
Add toml formatter to dprint

### PR DESCRIPTION
**Pull request by sujuka99**

### CODE REVIEW

File: dprint.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "$schema": "https://dprint.dev/schemas/v0.json",
   "incremental": true,
   "markdown": {},
+  "toml": {},
   "includes": [
     "**/*.{md}"
```
Comment (disrupted) [inline dprint.json:7]: `toml` file extension needs to be added

**Review by disrupted — DISMISSED** [review]

**Comment by disrupted** [comment]
please also rename the dprint step in the CI since it's no longer specific to markdown files

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Install docs dependencies
         run: poetry install --with docs
       
-      - name: Check markdown formatting
+      - name: Check markdown, css, toml formatting
```
Comment (raminqaf) [inline .github/workflows/ci.yaml:73]: You still haven't add the CSS formatter
```suggestion
      - name: Check markdown, toml formatting
```

**Review by raminqaf — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (sujuka99) [reply .github/workflows/ci.yaml:73]: This PR only concerns `TOML`, the [Malva PR](https://github.com/bakdata/kpops/pull/385) adds the `css` extension

**Review by irux — DISMISSED** [review]
LGTM

**Review by raminqaf — APPROVED** [review]

**Review by irux — APPROVED** [review]

**Pull request merged**